extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -219,6 +221,32 @@ class AmpAccordion extends AMP.BaseElement {
     }
   }
 
+  /**
```

Comment:
This code is exactly the same short of the name. You can also do: 
```
triggerEvent_(name, section);
```
for brevity, but I'm not very opinionated either way.